infra/base-images/base-runner/bad_build_check

```diff
@@ -219,19 +236,33 @@ function check_seed_corpus {
   if [ $? -ne 0 ]; then
     echo "BAD BUILD: $FUZZER has a crashing input in its seed corpus:"
     cat $FUZZER_OUTPUT
+    return 1
   fi
+
+  return 0
 }
 
 
 function main {
   local FUZZER=$1
+  local checks_failed=0
+  local result=0
 
   check_instrumentation $FUZZER
+  result=$?
+  checks_failed=$(( $checks_failed + $result ))
```

Comment:
Do we need to continue if any one step failed ? maybe we just return ?

Reply:
Yeah, the original intent was to test things as much as possible so that if there were two problems we would show both of them in the log, rather then reporting and fixing them one by one,